src/ontology/mondo-edit.obo

```diff
@@ -513752,6 +513752,31 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7099" xsd:anyURI
 
+[Term]
+id: MONDO:0100535
+name: clonal hematopoiesis
+def: "Cellular proliferation of hematopoietic cells where a substantial proportion of the cells is derived from a single hematopoietic stem cell lineage." [NCIT:C162188, PMID:34215849]
```

Comment:
to make it clear that we are talking about the condition, should we start the definition with 
"A precancerous condition characterized by ..."

Reply:
good suggestion, fixed